streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -79,6 +79,7 @@ void createTasks(final Collection<TopicPartition> assignment) {
         active.closeNonAssignedSuspendedTasks(assignedActiveTasks);
         addStreamTasks(assignment);
         addStandbyTasks();
+        changelogReader.refreshChangelogInfo();
```

Comment:
Do we need this call? Can't we just do it as the first thing in `StoreChangelogReader#initialize`?

Reply:
Yup, if we do this at the beginning of `initialize` this can be removed.